js/wallet.js

```diff
@@ -5,25 +5,89 @@
 'use strict'
 
 const Immutable = require('immutable')
-
 const sinon = require('sinon')
+
 const Wallet = require('../abs/wallet')
 const {request, roundtrip} = require('../lib/request')
+const settings = require('../browser-laptop/js/constants/settings')
+const responses = require('../lib/data/responses.json')
 
 const defaultAppState = Immutable.fromJS({
   cache: {
     ledgerVideos: {}
   },
-  ledger: {}
+  ledger: {
+    about: {}
+  },
+  about: {
+    preferences: {}
+  }
 })
 
 class JS extends Wallet {
   constructor () {
     super()
     this.ledger = null
+    this.minimumVisits = 1
+    this.minimumVisitTime = 5000
```

Comment:
Let's change this to 8000, this is our default in the browser

Reply:
++